FieldOpt/CMakeLists.txt

```diff
@@ -1,36 +1,51 @@
 cmake_minimum_required(VERSION 2.8)
 project(FieldOpt)
 
+# General config =======================================================
+# CMAKE_MODULE_PATH: Commands like include() and find_package() search
+# for files in directories listed by this variable before checking the
+# default modules that come with CMake.
+set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR})
+include(GetSysInfo.cmake)
+include(MaurConfig.cmake)
+
+# Compile options ======================================================
+set (CMAKE_CXX_STANDARD 11)
+
 # Build dirs ===========================================================
 set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 
+
```

Comment:
It is not strictly obselete, it is for the `make install` command. However, we don't really use that, I would much rather set up a Docker configuration, so I agree that it should be removed.